pkg/obfuscate/credit_cards.go

```diff
@@ -54,7 +54,7 @@ func (o *Obfuscator) ObfuscateCreditCardNumber(key, val string) string {
 		// these tags are known to not be credit card numbers
 		return val
 	}
-	if strings.HasPrefix(key, "_dd") {
+	if strings.HasPrefix(key, "_") {
```

Comment:
Are they really all hidden in the front end? Is it a contract (even informal) that they will maintain? Should we add a comment in the UI code where they hide things to let them know we use this on our end?

Reply:
Yep it's hidden in the front end, I confirmed with them internally and this feature is already relied on by a bunch of tags we use internally like _sampling_priority_v1 and _top_level